clients/drcachesim/tests/filter-no-i.templatex

```diff
@@ -21,7 +21,7 @@ Total counts:
            0 total physical address unavailable markers
```

Comment:
> Fixes TID tracking in basic counts which is broken for data-only traces.
> This was changed recently by PR https://github.com/DynamoRIO/dynamorio/pull/6022.

This description confused me into thinking this was broken prior to PR #6022 but looking at that PR it seems to directly cause the breakage.  Please reword to make it clearer that this was not a pre-existing problem.